packages/lib-classifier/src/components/Classifier/components/Layout/components/CenteredLayout/CenteredLayout.js

```diff
@@ -1,94 +1,98 @@
-import { useContext } from 'react'
 import styled, { css } from 'styled-components'
-import { Box, ResponsiveContext } from 'grommet'
+import { Box, Grid } from 'grommet'
 
 import Banners from '@components/Classifier/components/Banners'
 import FeedbackModal from '@components/Classifier/components/Feedback'
-import ImageToolbar from '@components/Classifier/components/ImageToolbar'
 import MetaTools from '@components/Classifier/components/MetaTools'
 import QuickTalk from '@components/Classifier/components/QuickTalk'
 import SubjectViewer from '@components/Classifier/components/SubjectViewer'
 import TaskArea from '@components/Classifier/components/TaskArea'
 import FieldGuide from '@components/Classifier/components/FieldGuide'
+import {
+  ViewerGrid,
+  StyledSubjectContainer,
+  StyledImageToolbarContainer,
+  StyledImageToolbar,
+  StyledTaskAreaContainer,
+  StyledTaskArea
+} from '../shared/StyledContainers'
 
-export const Relative = styled(Box)`
-  position: relative; // Used for QuickTalk and FeedbackModal positioning
-`
-
-const StickySubjectViewer = styled(Box)`
-  ${props =>
-    props.size !== 'small' &&
-    css`
-      position: sticky;
-      top: 10px;
-    `}
-`
+const ContainerGrid = styled(Grid)`
+  position: relative;
+  grid-gap: 1.875rem;
+  grid-template-areas: 'viewer task';
+  grid-template-columns: auto ${props => (props.hasSurveyTask ? '33.75rem' : '25rem')};
```

Comment:
I think the `grid-template-columns` value and any following line (CenteredLayout -> `justify-content: center`, MaxWidth -> `margin: auto`, and NoMaxWidth nothing) are they key differences between layouts? Could add a related/similar comment to each layout accordingly if that would be helpful.